docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -262,24 +282,24 @@ To check the version of the ``ansible`` package that has been installed:
 Adding Ansible command shell completion
 =======================================
 
-You can add shell completion of the Ansible command line utilities by installing an optional dependency called ``argcomplete``. ``argcomplete`` supports bash, and has limited support for zsh and tcsh.
+You can add shell completion of the Ansible command line utilities by installing an optional dependency called ``argcomplete``, which supports bash, and has limited support for zsh and tcsh.
```

Comment:
This change makes the sentence less readable. Longer sentences are harder to understand, especially by non-native English speakers. The original is two sentences for this reason.

Reply:
I agree. The reason i made it was that the repetition (esp. as it's lowercase) didn't look good. I've replaced it with just `it`. PTAL